src/style.css

```diff
@@ -0,0 +1,84 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.search {
+  display: flex;
+  box-sizing: border-box;
+  font-weight: 300;
+  position: relative;
+  margin-top: 20px;
+}
+
+.small {
+  background-image: url(images/Search.svg);
+  background-position-x: 13px;
+  background-position-y: 14px;
+  background-repeat: no-repeat;
+  background-size: 11px;
+  position: relative;
+  padding-left: 33px;
+  height: 42px;
+  width: 100%;
+  margin: auto;
+  font-family: inherit;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.big {
+  background-image: url(images/Search.svg);
+  background-position-x: 26px;
+  background-position-y: 25px;
+  background-repeat: no-repeat;
+  background-size: 19px;
+  margin-top: auto;
+  padding-left: 62px;
+  position: relative;
+  font-family: inherit;
+  height: 70px;
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 16px;
+  line-height: 22px;
+}
```

Comment:
1. follow the checklist for creating class names (point 2)
2. DRY, you shouldn't repeat styles